keybot/win_jenkins/test/test_jenkins.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
no underscores in package names..

if this is a utility maybe don't call it test... maybe call it a main package..e.g. `slackbot/jenkins/main/`